terminado/management.py

```diff
@@ -155,23 +167,36 @@ def new_terminal(self, **kwargs):
         argv = options['shell_command']
         env = self.make_term_env(**options)
         pty = PtyProcessUnicode.spawn(argv, env=env, cwd=options.get('cwd', None))
+        if os.name == 'nt':
```

Comment:
This will go away pending a new release of [pywinpty](https://github.com/spyder-ide/pywinpty/pull/65)